metacat-connector-hive/src/test/groovy/com/netflix/metacat/connector/hive/HiveConnectorFastPartitionSpec.groovy

```diff
@@ -49,122 +48,51 @@ class HiveConnectorFastPartitionSpec extends Specification {
     @Shared
     DirectSqlSavePartition directSqlSavePartition = Mock(DirectSqlSavePartition)
     @Shared
-    ConnectorRequestContext connectorContext = new ConnectorRequestContext(1, null);
```

Comment:
Did we remove test cases?

Reply:
Nope, this change is to consolidate them into one test.